### DISCUSSION TITLE
MS1 quantification using DIA-NN

### DISCUSSION THREAD

**Discussion by babypolarbear0412** [vdemichev/DiaNN#1069] [Q&A]
Hi vdem!!

I collected BoxCar DIA data using OE480 (orbitrap instrument).

I want to perform MS1 quantitation to maximize the advantages of BoxCar scan (MS1).

is it possible to perform MS1 quantitation using DIA-NN?

Sincerely,

Dongyoon,

**Comment by vdemichev** [comment]
Hi Dongyoon,

Yes, you get MS1 precursor quantities in the .parquet main report of DIA-NN 1.9. Can then calculate protein quantities based on those using the iq R package. But when analysing with QuantUMS, which is enabled by default, Precursor.Normalised and PG.MaxLFQ already integrate both MS1 and MS2, so will likely be better.

Best,
Vadim

**Comment by babypolarbear0412** [comment]
Thank you for your reply!

Best,
Dongyoon
